hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/AbstractEvictableRecordStore.java

```diff
@@ -426,6 +421,13 @@ protected void mergeRecordExpiration(Record record, EntryView mergingEntry) {
     }
 
     /**
+     * Returns value of removed record.
```

Comment:
This is not proper javadoc. You only explain the return value, but not the actual behavior.